### PR TITLE
test: add a test for BaseWindow.hookWindowMessage

### PR DESCRIPTION
**Pull request by nornagon**
#### Description of Change
There was no test for this and it [broke](https://github.com/electron/electron/pull/24716).

#### Checklist

- [x] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [x] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] This is **NOT A BREAKING CHANGE**. Breaking changes may not be merged to master until 11-x-y is branched.

#### Release Notes

Notes: none

### CODE REVIEW

File: spec-main/api-browser-window-spec.ts [hunk 1]
```diff
@@ -4288,4 +4288,20 @@ describe('BrowserWindow module', () => {
       });
     });
   });
+
+  ifdescribe(process.platform === 'win32')('hookWindowMessage', () => {
+    it('can hook a window message', async () => {
+      const w = new BrowserWindow({ show: true, width: 256, height: 256 });
```
Comment (deepak1556) [inline spec-main/api-browser-window-spec.ts:4294]: is the window cleaned up after this run ?

Comment (nornagon) [reply spec-main/api-browser-window-spec.ts:4294]: sorry, meant to mark this [wip]. i plan to add cleanup + also test `unhook`.

Comment (deepak1556) [reply spec-main/api-browser-window-spec.ts:4294]: got it, thanks! will dismiss my review.

**Comment by nornagon** [comment]
@deepak1556 un-wip now, tests should be more complete.

**Comment by nornagon** [comment]
Hm, I'm surprised that these tests are passing, given https://github.com/electron/electron/pull/24716 has not yet merged.

**Pull request closed without merging**